dist/vuera.cjs.js

```diff
@@ -341,7 +341,8 @@ var makeReactContainer = function makeReactContainer(Component) {
 var ReactWrapper = {
   props: ['component', 'passedProps'],
   render: function render(createElement) {
-    return createElement('div', { ref: 'react' });
+    this.refString = 'react-' + Math.random().toString();
```

Comment:
can use uuid for generate unique references ?

Reply:
There's no need, your algorithm is good enough. It's just that we should not generate this on every `render`.